### PR TITLE
Add information about movie extras

### PR DESCRIPTION
**Pull request by kayila**
Added information about movie extras and how to store them.

### CODE REVIEW

File: general/server/media/movies.md [hunk 1]
```diff
@@ -50,3 +50,98 @@ If labels are not added to the end of filenames, as shown above, each file will
...
 > To group media manually, long-click or right-click media to highlight then select additional media to merge. Use the new bar that appears to 'Group Versions'.
+
+## Movie extras
+
+Movie extras can include deleted scenes, interviews, and other various things that you would want to include alongside your movie. Jellyfin supports several different methods of adding these files.
+
+### Extras folders
+
+One of the cleanest ways of adding extras is to place them in subfolders
+within your movie folder.
```
Comment (h1dden-da3m0n) [inline general/server/media/movies.md:61]:
```suggestion
One of the cleanest ways of adding extras is to place them in subfolders within your movie folder.
```

**Review by h1dden-da3m0n — APPROVED** [review]

**Review by Artiume — APPROVED** [review]

**Comment by dkanada** [comment]
These are excellent additions to the media naming rules, thanks for the improvements!

File: general/server/media/movies.md [hunk 2]
```diff
@@ -61,3 +58,100 @@ Movie versions are presented in an alphabetically sorted list. An exception appl
...
+    ├── theme.mp3
+    └── trailer.mp4
+```
+
+### File Suffix
+
+If you would rather keep everything in a single folder, you can append special suffixes to the filename which Jellyfin picks up and uses to identify the file as an extra. Note that, with a few noted exceptions, these suffexes **DO NOT** contain any spaces.
+
+Supported file suffexes are:
+
```
Comment (dkanada) [inline general/server/media/movies.md:123]:
```suggestion
```

File: general/server/media/movies.md [hunk 3]
```diff
@@ -61,3 +58,100 @@ Movie versions are presented in an alphabetically sorted list. An exception appl
...
+    ├── sample.mp4
+    ├── theme.mp3
+    └── trailer.mp4
+```
+
+### File Suffix
+
+If you would rather keep everything in a single folder, you can append special suffixes to the filename which Jellyfin picks up and uses to identify the file as an extra. Note that, with a few noted exceptions, these suffexes **DO NOT** contain any spaces.
+
+Supported file suffexes are:
```
Comment (dkanada) [inline general/server/media/movies.md:122]:
```suggestion
```

**Review by dkanada — APPROVED** [review]

**Pull request merged**